src/main/java/com/work/management/service/project/ProjectService.java

```diff
@@ -0,0 +1,5 @@
+package com.work.management.service.project;
+
+public interface ProjectService<ProjectDto, Project> {
+    void save(ProjectDto document);
```

Comment:
Same here remove <ProjectDto,Project> and rename document to projectDto... every name should be meaningful..it should represent what it is.